CHANGELOG.md

```diff
@@ -1,19 +1,56 @@
+- GPS time vs. position graph changes with each program run [177383703](https://www.pivotaltracker.com/story/show/177383703)
+
+### Features/Improvements
+- Update browser tab name [175747402](https://www.pivotaltracker.com/story/show/175747402)
+- SEISMIC: Axes Display Equal Intervals on Position over Time Graph [176707032](https://www.pivotaltracker.com/story/show/176707032)
+- SEISMIC: Add (W, N) to Lat-Long tool info boxes [176787959](https://www.pivotaltracker.com/story/show/176787959)
+- Authorable Default Model Tab Display [175557888](https://www.pivotaltracker.com/story/show/175557888)
+- SEISMIC: Gridlines on position/location graph [174846693](https://www.pivotaltracker.com/story/show/174846693)
+- Blocks Always Visible in LARA [176593685](https://www.pivotaltracker.com/story/show/176593685)
+- Display Tool buttons on Map based on Unit [177292659](https://www.pivotaltracker.com/story/show/177292659)
+- SEISMIC: Lat-Long Tool UX [177204160](https://www.pivotaltracker.com/story/show/177204160)
+- SEISMIC: Station Markers in Deformation Sim [177208787](https://www.pivotaltracker.com/story/show/177208787)
+- SEISMIC: Update Right Panel Tab Colors [177208579](https://www.pivotaltracker.com/story/show/177208579)
+- SEISMIC: Fault Line Label in Deformation Sim [177077410](https://www.pivotaltracker.com/story/show/177077410)
+- SEISMIC: Rename Lat-Long Tool to Set region [177285776](https://www.pivotaltracker.com/story/show/177285776)
+- SEISMIC: Update Deformation Text Positions [177325665](https://www.pivotaltracker.com/story/show/177325665)
+- TEPHRA: Ruler Tool Styling [177204431](https://www.pivotaltracker.com/story/show/177204431)
+- (Seismic) Delete GPS station WMAP [177275272](https://www.pivotaltracker.com/story/show/177275272)
+- SEISMIC: Update GPS Station Table Styling [177289909](https://www.pivotaltracker.com/story/show/177289909)
+- SEISMIC: Update Key Styling [177288598](https://www.pivotaltracker.com/story/show/177288598)
+- TEPHRA: Update Key Styling [177288607](https://www.pivotaltracker.com/story/show/177288607)
+- SEISMIC: Update button styling [177430008](https://www.pivotaltracker.com/story/show/177430008)
+- SEISMIC: Plate Colors in Deformation Sim [177208666](https://www.pivotaltracker.com/story/show/177208666)
+- SEISMIC: Set Point Tool [177256960](#177256960)
+- SEISMIC: Plate Motion Widgets in Deformation Sim [174846688](https://www.pivotaltracker.com/story/show/174846688)
+- SEISMIC: GPS Location vs Time Graph Styling [175037328](https://www.pivotaltracker.com/story/show/175037328)
+- SEISMIC: Explore Direction Widget [176787693](https://www.pivotaltracker.com/story/show/176787693)
+- SEISMIC: Make Explore Direction Widget Movable [177429998](https://www.pivotaltracker.com/story/show/177429998)
+- TEPHRA: Make tephra distribution colors semi-transparent [169984469](https://www.pivotaltracker.com/story/show/169984469)
+
 ## Version 2.2.3 - release January 20, 2020
```

Comment:
```suggestion
## Version 2.2.3 - release January 20, 2021
```

Reply:
Ah, thanks!  A bunch of these had the wrong date.  Looks like I missed one.